packages/@okta/vuepress-site/docs/release-notes/2022/index.md

```diff
@@ -4,6 +4,22 @@ title: Okta API Products release notes 2022
 
 ## July
 
+### Weekly release 2022.07.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Dynamic issuer mode for identity providers](#dynamic-issuer-mode-for-identity-providers)                    | July 13, 2022            |
+| [Bug fixed in 2022.07.1](#bug-fixed-in-2022-07-1)                        | July 13, 2022            |
+
+#### Dynamic issuer mode for identity providers
+
+You can configure the dynamic issuer mode for an identity provider using the Identity Provider API. When you set `issuerMode` to `DYNAMIC`, Okta uses the domain from the `authorizeURI` as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
+
+#### Bug fixed in 2022.07.1
+
+When the `/api/v1/users/{userId}/roles` or
+`/api/v1/groups/{groupId}/roles` were called to assign a custom role and resource set to a user or group, and those assignments already existed, the calls didn’t receive an HTTP 409 Conflict error. (OKTA-507683)
```

Comment:
```suggestion
`/api/v1/groups/${groupId}/roles` endpoints were called to assign a custom role and resource set to a user or group, and those assignments already existed, the calls didn’t receive an HTTP 409 Conflict error. (OKTA-507683)
```
Not sure if we are still using `${var}` notation?

Reply:
Updated locally. Yes, we're still using the default `${var}` notation for now. No response from Dan's dev survey as far as I know.